src/test/java/plugins/JobDslPluginTest.java

```diff
@@ -470,10 +472,47 @@ public void should_fail_on_missing_plugin() {
                          "}");
         seedJob.save();
         Build build = seedJob.scheduleBuild().shouldBeUnstable();
+        assertThat(build.getConsole(), containsString("plugin 'chucknorris' needs to be installed"));
+
+        seedJob.configure(() -> jobDsl.setFailOnMissingPlugin(true));
+        build = seedJob.scheduleBuild().shouldFail();
+        assertThat(build.getConsole(), containsString("plugin 'chucknorris' needs to be installed"));
```

Comment:
just change the assert to pass on Java 11

Reply:
Is it me or the diff is incredibly confusing here? It appears you have fixed the test by _adding_ assertions while you have _modified_ lines instead.

[1] https://github.com/jenkinsci/acceptance-test-harness/blob/7e8c9b20d25e420c37940313f8243e21c86be1bb/src/test/java/plugins/JobDslPluginTest.java#L472-L477

Reply:
Yes, I saw the same. It's due to the change on the _assertThat_ that the diff becomes crazy. Due to that, I added some comments to the PR :man_shrugging:

Reply:
Curious: wouldn't adding some `assumeTrue(isOnJava8OrLess())`, instead of adding a new ignored test, achieve some similar outcome?

I think it's generally better to try and avoid adding more ignored tests, WDYT?

Reply:
The test is still valid with this assert and we still want the failing test on Java 11 for using it to fix the loss of information in the log (if it's really possible to fix).That's the reason I didn't use an approach like you suggest.